cmd/juju/caas/add.go

```diff
@@ -88,40 +105,75 @@ func (c *AddCAASCommand) SetFlags(f *gnuflag.FlagSet) {
 // Init populates the command with the args from the command line.
 func (c *AddCAASCommand) Init(args []string) (err error) {
 	if len(args) > 0 {
-		c.CAASType = args[0]
+		c.caasType = args[0]
 	}
 	if len(args) > 1 {
-		c.CAASName = args[1]
+		c.caasName = args[1]
 	}
 	if len(args) > 2 {
```

Comment:
This `if` is unnecessary. `return cmd.CheckEmpty(args[2:])` is all that's required.